apps/oxlint/src/lint.rs

```diff
@@ -423,6 +445,15 @@ mod test {
         assert_eq!(result.number_of_errors, 0);
     }
 
+    #[test]
+    fn oxlint_config_auto_detection() {
+        let args = &["debugger.js"];
+        let result = test_with_cwd("fixtures/auto_config_detection", args);
+        assert_eq!(result.number_of_files, 1);
+        assert_eq!(result.number_of_warnings, 0);
+        assert_eq!(result.number_of_errors, 1);
```

Comment:
isn't debugger on by default since it's in `correctness` ?
so if it failed to pick up the config, it would still return 1 here

Reply:
It will output it as an warning not an error. 
Deleting the `oxlint.json` the assertion should be:
```
assert_eq!(result.number_of_warnings, 1);
assert_eq!(result.number_of_errors, 0);
```

Reply:
gotcha - thanks!